roles/jd-server/src/main.rs

```diff
@@ -118,9 +118,20 @@ async fn main() {
                 let updated_mempool =
                     mempool::JDsMempool::update_mempool(mempool_cloned_.clone()).await;
                 if let Err(err) = updated_mempool {
-                    error!("{:?}", err);
-                    error!("Unable to connect to Template Provider (possible reasons: not fully synced, down)");
-                    handle_result!(sender_clone, Err(err));
+                    warn!("{:?}", err);
+                    match err {
+                        mempool::JdsMempoolError::EmptyMempool => {
+                            warn!("Template Provider is running, but its mempool is empty (possible reasons: you're testing in testnet, signet, or regtest)")
```

Comment:
This works, but it's logged every second.

Reply:
Rate is determined by the `[mempool_update_timeout]` parameter set in config

Reply:
Ah, now I remember you already suggested a change in this parameter name!
Don't know where to find that suggestion btw :D

Reply:
Polling every second is fine, but the warning should just appear once, otherwise the log becomes too noisy.

Reply:
you could log it only once using something like `let mut empty_mempool_logged = false`

Reply:
Or you could make it more noisy on mainnet only.